spec/requests/api/orchestration_template_spec.rb

```diff
@@ -125,4 +125,70 @@
       expect(OrchestrationTemplate.exists?(hot.id)).to be_falsey
     end
   end
+
+  context 'orchestration template copy' do
+    it 'copy template with the same content' do
```

Comment:
Can you make these fully descriptive and full sentences (including the "it") so something like "it will not copy a template with no attributes specified" or similar...